### PR TITLE
Mimes Scream Silently

### PR DESCRIPTION
**Pull request by AwkwardDryad**
[FEATURE]
## About the PR 
- Mimes can no longer audibly scream. Instead, they will visibly perform one of a few special emotes.
(this applies to anyone with the mute mutation)
- Additionally, mimes have a 20% chance to spawn in with the noir mutation! 

## Why's this needed? 
- mimes are surprisingly noisy.

## Changelog

```
(u)AwkwardDryad:
(+)Recent reports indicate that mimes are quiter than ever. They've even seemed darker and paler on occasion...
```

### CODE REVIEW

**Comment by AwkwardDryad** [comment]
note: the chance for the emote text to be "" is 100% intentional. It's beautiful and hilarious.

**Comment by comradef191** [comment]
this is so dumb
i fucking love it
best PR of January so far

File: code/mob/living/carbon/human/procs/emote.dm [hunk 1]
```diff
@@ -41,7 +41,15 @@
 			// most commonly used emotes first for minor performance improvements
 			if ("scream")
 				if (src.emote_check(voluntary, 50))
-					if (!muzzled)
+					if(src.bioHolder && src.bioHolder.HasEffect("mute"))
```
Comment (pali6) [inline code/mob/living/carbon/human/procs/emote.dm:44]:
```suggestion
					if(src.bioHolder?.HasEffect("mute"))
```

**Review by flrsh — CHANGES_REQUESTED** [review]
I think having a 1/5 chance of spawning with noir would put some people off of playing mime. I'd rather they didn't have this.

File: code/mob/living/carbon/human/procs/emote.dm (hunk 1 above)
Comment (AwkwardDryad) [reply code/mob/living/carbon/human/procs/emote.dm:44]: oooh i didnt know you could format things like that!

**Comment by AwkwardDryad** [comment]
> I think having a 1/5 chance of spawning with noir would put some people off of playing mime. I'd rather they didn't have this.

eeh? it just changes the character sprite color though. plus mimes already spawn pale white, the only real difference makes is that inhands are rendered in monochrome

**Comment by flrsh** [comment]
Oh, whoops, I totally confused it with the det's noir. Seems fine to me then!

**Comment by AwkwardDryad** [comment]
everything should be addressed now 🥰

File: code/mob/living/carbon/human/procs/emote.dm [hunk 2]
```diff
@@ -41,7 +41,15 @@
...
 			if ("scream")
 				if (src.emote_check(voluntary, 50))
-					if (!muzzled)
+					if(src.bioHolder?.HasEffect("mute"))
+						var/list/mimescreams = list("vibrates for a moment, then stops", "opens [his_or_her(src)] mouth incredibly wide, but no sound comes out",
+						"really wants to be noticed", "emits an audible silence","lets forth the silent echoes of an empty soul","huffs and puffs with all [his_or_her(src)] might, but can't seem to make a sound",
+						"unhinges [his_or_her(src)] maw to produce a deafening, roaring lack of any noise whatsoever","flails desperately","")
+						var/pre_message = "[pick(mimescreams)]..."
+						message = "<B>[src]</B> [pre_message]"
+						maptext_out = "<i>[pre_message]</i>"
```
Comment (flrsh) [inline code/mob/living/carbon/human/procs/emote.dm:50]:
```suggestion
						var/pre_message = "[pick("vibrates for a moment, then stops", "opens [his_or_her(src)] mouth incredibly wide, but no sound comes out",
						"really wants to be noticed", "emits an audible silence","lets forth the silent echoes of an empty soul","huffs and puffs with all [his_or_her(src)] might, but can't seem to make a sound",
						"unhinges [his_or_her(src)] maw to produce a deafening, roaring lack of any noise whatsoever","flails desperately","")]..."
						message = "<B>[src]</B> [pre_message]"
						maptext_out = "<i>[pre_message]</i>"
```

**Review by flrsh — APPROVED** [review]

**Review by ZeWaka — APPROVED** [review]

**Review by pali6 — APPROVED** [review]

**Pull request merged**